shell/platform/android/io/flutter/embedding/android/FlutterImageView.java

```diff
@@ -197,7 +197,7 @@ public boolean acquireLatestImage() {
     // or some special Android devices so the calls to `invalidate()` queued up
     // until the device produces a new frame.
     // 3. While the engine will also stop producing frames, there is a race condition.
-    final Image newImage = imageReader.acquireLatestImage();
+    final Image newImage = imageReader.acquireNextImage();
```

Comment:
Need to check if we use this in other places. We want to use nextIMage because we want to block on scheduling.